aee.js

```diff
@@ -72,3 +72,13 @@ Array.prototype.count = function (spec) {
   }
   return result
 }
+
+Array.prototype.pluck = function (property) {
+  if (property === undefined) throw new ReferenceError('property is undefined')
+  const len = this.length
+  let result = []
+  for (let i = 0; i < len; i += 1) {
+    result.push(this[i] && this[i].hasOwnProperty(property) ? this[i][property] : null)
```

Comment:
You decide what to return, but whatever your choice is don't forget to properly document it. Remember that if it's properly documented it's not a bug, it's a feature 😉